src/state/slices/tradeQuoteSlice/selectors.ts

```diff
@@ -241,6 +241,23 @@ export const selectActiveQuote: Selector<ReduxState, TradeQuote | undefined> =
     },
   )
 
+export const selectQuoteSlippageTolerancePercentageDecimal: Selector<
```

Comment:
Not consumed outside of this module, remove const or consolidate me into `selectQuoteSlippageTolerancePercentage`

Reply:
Removed export in https://github.com/shapeshift/web/pull/7654/commits/a3308ec5f5f6278a78ddf90756835c6445d610be